src/components/content/content.test.js

```diff
@@ -0,0 +1,80 @@
+import * as React from "react";
+import Content from "./content.component";
+import CypressMountWithProviders from "../../../cypress/support/component-helper/cypress-mount";
+
+import {
+  contentTitle,
+  contentBody,
+  contentPreview,
+} from "../../../cypress/locators/content/index";
+
+context("Tests for Content component", () => {
+  describe("should check Content component properties", () => {
+    it.each(["mp150ú¿¡üßä", "!@#$%^*()_+-=~[];:.,?{}&\"'<>"])(
+      "should check %s as children for Content component",
+      (children) => {
+        CypressMountWithProviders(<Content title="Title">{children}</Content>);
+        contentBody().should("have.text", children);
+      }
+    );
+
+    it.each([
+      ["mp150ú¿¡üßä", "primary"],
+      ["!@#$%^*()_+-=~[];:.,?{}&\"'<>", "secondary"],
+    ])("should check %s as title for Content component", (title, variant) => {
+      CypressMountWithProviders(
+        <Content title={title} variant={variant}>
+          This is an example of some content
+        </Content>
+      );
+      contentTitle().should("have.text", title);
+    });
+
+    it("should check inline parameter is enabled for Content component", () => {
+      CypressMountWithProviders(
+        <Content title="Title" inline>
+          This is an example of some content
+        </Content>
+      );
+      contentTitle().should("have.css", "display", "inline-block");
+    });
+
+    it.each(["left", "center", "right"])(
+      "should check %s alignment for Content component",
+      (align) => {
+        CypressMountWithProviders(
+          <Content title="Title" align={align}>
+            This is an example of some content
+          </Content>
+        );
+        contentTitle().should("have.css", "text-align", align);
+      }
+    );
+
+    it.each([0, 1350])(
+      "should check %s width for Content component",
+      (width) => {
+        CypressMountWithProviders(
+          <Content title="Title" titleWidth={`${width}px`}>
+            This is an example of some content
+          </Content>
+        );
+        contentTitle()
+          .should("have.css", "width")
+          .and("contains", `${width}px`);
+      }
+    );
+
+    it("should check Content component has no bodyFullWidth parameter", () => {
+      CypressMountWithProviders(
+        <Content title="Title" bodyFullWidth={false}>
```

Comment:
`bodyFullWidth={false}` - it is set to `false` by default - we need to check if will be set to `true`